packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -38,7 +39,7 @@ const Breakouts = WebexPlugin.extend({
     mainLocusUrl: 'string', // the locus url of the main session
     groups: 'array', // appears when create breakouts
   },
-
+  isCallPreassignments: false,
```

Comment:
This parameter is mainly to control the pre-assign API to only call once. No matter how complicated the user behavior , we only need to call once after the user’s identity is clarified. These cases can be triggered multiple times. For example, the user upgrades and downgrades repeatedly , switch the current breakout enable or disable, so the life cycle of this parameter is the same as the current breakout session. When the breakout session is reset, then we can call it again.
The main purpose is to reduce the useless API on the network. This list is set when the meeting was scheduled. We only need to take it when it is time to take it, and take it once.

Reply:
Ok, thanks for the explanation. Can you change it to use the model attributes rather than add an attribute to the class? i.e. can you put the variable in to props? I also think a name like `shouldFetchPreassignments` would be a better name to describe what it is doing.